pkg/action/action.go

```diff
@@ -19,19 +19,23 @@ package action
 import (
 	"path"
 	"regexp"
+	"sync"
 	"time"
 
 	"github.com/pkg/errors"
 	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/client-go/discovery"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 
 	"helm.sh/helm/internal/experimental/registry"
 	"helm.sh/helm/pkg/chartutil"
+	"helm.sh/helm/pkg/cli"
```

Comment:
Helm moved to Go modules. That means the major version needs to be in the import path. This applies when something has a major version of 2 or higher. So, the helm import paths need to have a `/v3` in them. For example,

`"helm.sh/helm/pkg/cli"` -> `"helm.sh/helm/v3pkg/cli"`

Reply:
I rebased with the latest. Should be good now.